src/components/dialogs/network-modifications/limit-sets/limit-sets-tabular-modification-form.tsx

```diff
@@ -0,0 +1,268 @@
+                    );
+                    return transformedHeader ?? header;
+                },
+                transform: (value) => transformIfFrenchNumber(value, language),
+            });
+        }
+    }, [clearErrors, getValues, handleComplete, intl, selectedFile, selectedFileError, setValue, language]);
+
+    const typesOptions = useMemo(() => {
+        //only available types for tabular modification
+        return Object.keys(LIMIT_SETS_TABULAR_MODIFICATION_EQUIPMENTS).filter(
+            (type) => EQUIPMENT_TYPES[type as keyof typeof EQUIPMENT_TYPES]
+        );
+    }, []);
+
+    const handleChange = useCallback(() => {
+        setTypeChangedTrigger(!typeChangedTrigger);
+        clearErrors(MODIFICATIONS_TABLE);
+        setValue(MODIFICATIONS_TABLE, []);
+    }, [clearErrors, setValue, typeChangedTrigger]);
+
+    const equipmentTypeField = (
+        <AutocompleteInput
+            name={TYPE}
+            label="Type"
+            options={typesOptions}
+            onChangeCallback={handleChange}
+            getOptionLabel={(option: any) => getTypeLabel(option as string)}
+            size={'small'}
+            formProps={{ variant: 'filled' }}
+        />
+    );
+
+    const defaultColDef = useMemo(
+        () => ({
+            sortable: true,
+            resizable: false,
+            lockPinned: true,
+            wrapHeaderText: true,
+            autoHeaderHeight: true,
+            cellRenderer: DefaultCellRenderer,
+        }),
+        []
+    );
+
+    const columnDefs = useMemo<ColDef[]>(
+        () =>
+            csvColumns.map(({ id, name, index }) => ({
+                field: id,
+                pinned: id === EQUIPMENT_ID ? true : undefined,
+                headerName: `${intl.formatMessage({ id: name ?? id })} ${index ?? ''}`,
+                cellRenderer: DefaultCellRenderer,
+            })),
+        [csvColumns, intl]
+    );
+
+    return (
+        <Grid container spacing={2} direction={'row'}>
+            <Grid container item spacing={2} alignItems={'center'}>
+                <GridItem size={4}>{equipmentTypeField}</GridItem>
+                <Grid item>{FileField}</Grid>
+            </Grid>
+            <Grid container item spacing={2}>
+                <Grid item>
+                    <IntegerInput name={AMOUNT_TEMPORARY_LIMITS} label={'amountTemporaryLimits'} />
+                </Grid>
+                <GridItem>
+                    <CsvDownloader
+                        columns={csvColumns}
+                        datas={commentLines}
+                        filename={equipmentType + '_modification_template'}
+                        disabled={!csvColumns}
+                        separator={language === LANG_FRENCH ? ';' : ','}
+                    >
+                        <Button variant="contained" disabled={!csvColumns}>
+                            <FormattedMessage id="GenerateSkeleton" />
```

Comment:
The generate skeleton action is triggered even if the mouse is outside right of the button, when clicking ...